scripts/analyze.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
this was like this before but I normally like to skip a line between the shebang and the rest of the lines